<!--REQUEST_ID=7cd2c88c-2efe-452d-a415-ed0d2e1f8317-->
Failure on aarch64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.48896&attempt_id=38ea0c52-107c-4f4d-8873-685b1e7f25ad)

Attempted: avldrums-lv2

<details><summary>Partial log (click to expand)</summary><p>

```
    lv2ttl/avldrums.gui.in >> build/avldrums.ttl
gcc -Ifluidsynth -I fluidsynth/fluidsynth -DHAVE_CONFIG_H -D DEFAULT_SOUNDFONT=\"\" -Wall -g -Wno-unused-function -DHAVE_LV2_1_8 -msse -msse2 -mfpmath=sse -ffast-math -fomit-frame-pointer -O3 -fno-finite-math-only -DNDEBUG -DVERSION="\"\"" `pkg-config --cflags lv2 glib-2.0` -fPIC -fvisibility=hidden  -std=c99 \
  -o build/avldrums.so src/avldrums.c fluidsynth/src/fluid_midi.c fluidsynth/src/fluid_adsr_env.c fluidsynth/src/fluid_chorus.c fluidsynth/src/fluid_iir_filter.c fluidsynth/src/fluid_lfo.c fluidsynth/src/fluid_rev.c fluidsynth/src/fluid_rvoice.c fluidsynth/src/fluid_rvoice_dsp.c fluidsynth/src/fluid_rvoice_event.c fluidsynth/src/fluid_rvoice_mixer.c fluidsynth/src/fluid_defsfont.c fluidsynth/src/fluid_chan.c fluidsynth/src/fluid_event.c fluidsynth/src/fluid_gen.c fluidsynth/src/fluid_mod.c fluidsynth/src/fluid_synth.c fluidsynth/src/fluid_tuning.c fluidsynth/src/fluid_voice.c fluidsynth/src/fluid_conv.c fluidsynth/src/fluid_hash.c fluidsynth/src/fluid_list.c fluidsynth/src/fluid_ringbuffer.c fluidsynth/src/fluid_settings.c fluidsynth/src/fluid_sys.c \
  -shared -Wl,-Bstatic -Wl,-Bdynamic -Wl,--as-needed -pthread  -lm `pkg-config  --libs glib-2.0`
gcc: error: unrecognized command line option '-msse'; did you mean '-fdse'?
gcc: error: unrecognized command line option '-msse2'
gcc: error: unrecognized command line option '-mfpmath=sse'
make: *** [Makefile:267: build/avldrums.so] Error 1
builder for '/nix/store/fjgx49gnga879y8v14gpsga4ryf4p1hd-avldrums.lv2-0.3.0.drv' failed with exit code 2
error: build of '/nix/store/fjgx49gnga879y8v14gpsga4ryf4p1hd-avldrums.lv2-0.3.0.drv' failed
```
</p></details>

--- a/pkgs/applications/audio/avldrums-lv2/default.nix
+++ b/pkgs/applications/audio/avldrums-lv2/default.nix
@@ -1,0 +1,30 @@
+{ stdenv, fetchFromGitHub, pkgconfig, pango, cairo, libGLU, lv2 }:
+
+stdenv.mkDerivation rec {
+  name = "${pname}-${version}";
+  pname = "avldrums.lv2";
+  version = "0.3.0";
+
+  src = fetchFromGitHub {
+    owner = "x42";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0w51gdshq2i5bix2x5l3g3gnycy84nlzf5sj0jkrw0zrnbk6ghwg";
+    fetchSubmodules = true;
+  };
+
+  installFlags = "PREFIX=$(out)";
+
+  nativeBuildInputs = [ pkgconfig ];
+  buildInputs = [
+    pango cairo libGLU lv2
+  ];
+
+  meta = with stdenv.lib; {
+    description = "Dedicated AVLDrumkits LV2 Plugin";
+    homepage    = http://x42-plugins.com/x42/x42-avldrums;
+    license     = licenses.gpl2;
+    maintainers = [ maintainers.magnetophon ];
+    platforms   = [ "i686-linux" "x86_64-linux" ];
+  };
+}

--- a/pkgs/top-level/all-packages.nix
+++ b/pkgs/top-level/all-packages.nix
@@ -577,6 +577,8 @@ with pkgs;
 
   avfs = callPackage ../tools/filesystems/avfs { };
 
+  avldrums-lv2 = callPackage ../applications/audio/avldrums-lv2 { };
+
   aws-iam-authenticator = callPackage ../tools/security/aws-iam-authenticator {};
 
   awscli = callPackage ../tools/admin/awscli { };
@@ -21123,7 +21125,7 @@ with pkgs;
   liblapackWithAtlas = liblapack;
 
   liblbfgs = callPackage ../development/libraries/science/math/liblbfgs { };
-  
+
   lrs = callPackage ../development/libraries/science/math/lrs { };
 
   m4ri = callPackage ../development/libraries/science/math/m4ri { };

